Library/Homebrew/cask/cask_loader.rb

```diff
@@ -103,6 +103,9 @@ def self.try_new(ref, warn: false)
         return if %w[.rb .json].exclude?(path.extname)
         return unless path.expand_path.exist?
 
+        return if Homebrew::EnvConfig.forbid_packages_from_paths? &&
+                  path.realpath.to_s.start_with?("#{Caskroom.path}/", "#{HOMEBREW_LIBRARY}/Taps/")
```

Comment:
Should this not be negated, to allow loading installed casks?

Reply:
Oops, that's what I get for messing with the syntax too much.

Thanks for spotting!